### PR TITLE
fix #2007

### LINKED ISSUE #2007: Author pages show stories
example: https://getnikola.com/authors/roberto-alsina.html → anything between 7.5.0 and 7.6.1 (most of which were written by me, but that’s not important)

cc @jjconti

### PR DESCRIPTION
**Pull request by jjconti**

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -1545,22 +1553,4 @@ Features
 * Image descriptions in galleries
 * Image exclusion in galleries
 * Special "draft" tag
-* Pretty code listings ("code galleries")
```
Comment (ralsina) [inline CHANGES.txt]: What happened here?

Comment (jjconti) [inline CHANGES.txt]: Weird. Let me review it.

Comment (jjconti) [inline CHANGES.txt]: PyCharm remove part of the file (?) and I didn't see if before commiting/pushing.

**Comment by ralsina** [comment]
Gracias por el arreglo Juanjo!

**Comment by jjconti** [comment]
:+1:

**Pull request merged**